// yallist/baked-ao-forked/src/card.js
import React from 'react'
import { Link } from 'react-router-dom'
import { makeStyles } from '@material-ui/core/styles'
import Card from '@material-ui/core/Card'
import CardActionArea from '@material-ui/core/CardActionArea'
import CardActions from '@material-ui/core/CardActions'
import CardContent from '@material-ui/core/CardContent'
import CardMedia from '@material-ui/core/CardMedia'
import Button from '@material-ui/core/Button'
import Typography from '@material-ui/core/Typography'

const useStyles = makeStyles({
  root: {
    width: '60%',
    maxWidth: 500,
    // width: 600,
  },
  div: {
    display: 'flex',
    padding: '50px',
  },
  desc: {
    color: 'black',
    width: '60%',
    marginLeft: '50px',
    fontFamily: '',
  },
  media: {
    paddingTop: '56.25%',
    height: 100,
  },
})

export default function MediaCard({ title, para, github, id }) {
  const classes = useStyles()

  return id % 2 == 0 ? (
    <div className={classes.div}>
      <Card id="rooter" className={classes.root}>
        <CardActionArea>
          <CardMedia
            className={classes.media}
            image="https://firebasestorage.googleapis.com/v0/b/confusion-1c185.appspot.com/o/images%2Fproject.jpg?alt=media&token=6785e11c-cc5f-46e1-bd9b-4d56c6e26abd"
            title={title}
          />
          <CardContent>
            <Typography variant="h5">{title}</Typography>
          </CardContent>
        </CardActionArea>
        <CardActions>
          <Button href={github} color="primary">
            Github
          </Button>
        </CardActions>
      </Card>
      <Typography variant="h5" id="decription" className={classes.desc} color="textSecondary">
        {para}
      </Typography>
    </div>
  ) : (
    <div style={{ flexDirection: 'row-reverse' }} className={classes.div}>
      <Card id="rooter" className={classes.root}>
        <CardActionArea>
          <CardMedia
            className={classes.media}
            image="https://firebasestorage.googleapis.com/v0/b/confusion-1c185.appspot.com/o/images%2Fproject.jpg?alt=media&token=6785e11c-cc5f-46e1-bd9b-4d56c6e26abd"
            title={title}
          />
          <CardContent>
            <Typography variant="h5">{title}</Typography>
          </CardContent>
        </CardActionArea>
        <CardActions>
          <Button href={github} color="primary">
            Github
          </Button>
        </CardActions>
      </Card>
      <Typography variant="h5" id="decription" className={classes.desc} color="textSecondary">
        {para}
      </Typography>
    </div>
  )
}
